### PR TITLE
feat(config): handle slab profile as input

### PR DESCRIPTION
**Pull request by soonum**
This requires the next version of Slab to be put in production in order to have a green CI.

### CODE REVIEW

**Comment by IceTDrinker** [comment]
ping me once the new slab is up/the tests are green

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,4 +74,25 @@ jobs:
           job-secret: ${{ secrets.JOB_SECRET }}
           region: eu-west-3
           label: ${{ steps.test-start.outputs.label }}
-          ec2-instance-id: ${{ steps.test-start.outputs.ec2-instance-id }}
```
Comment (IceTDrinker) [inline .github/workflows/ci.yml:77]: can you explain quickly why we get rid of it ? I'm not enough of a slab pro to see why we remove it

File: dist/index.js [hunk 2]
```diff
@@ -49801,12 +49832,23 @@ async function waitForInstance(taskId, taskName) {
...
+    }
+  } else {
+    payload = {
+      details: { custom_stop: { region: config.input.region } }
+    }
   }
+  payload.runner_name = runnerName
+  payload.action = 'terminate'
+  payload.sha = config.githubContext.sha
+  payload.git_ref = config.githubContext.ref
```
Comment (IceTDrinker) [inline dist/index.js:49850]: is it possible to have the const payload as before provided we fill details ahead of time or not ?

like

```js
let details

  if (config.input.profile) {
    detail= {
      profile: config.input.profile
    }
  } else {
      details = { custom_stop: { region: config.input.region } }
  }

  const payload = {
    "details": details,
    "runner_name": ...
  }
```

?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (soonum) [reply .github/workflows/ci.yml:77]: It was a leftover of machulav action. I thought at first we might need it to reference existing EC2 instances but it turns out that we don't.

**Review by IceTDrinker — APPROVED** [review]
Lesgo

**Pull request merged**